.deployment/roles/opencast/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+- name: stop opencast
+  service:
+    name: opencast
+    state: stopped
+  ignore_errors: true
+
+- name: remove previous deployment
+  ansible.builtin.file:
```

Comment:
What's the purpose of the full "path", i.e why the `ansible.builtin.`?

Reply:
Seems like Ansible is slowly switching (and also moving stuff from the core to external collections).
I guess you probably never need this fore builtin/core though, but it also does not hurt.